packages/grid/_modules_/grid/components/GridRow.tsx

```diff
@@ -63,7 +63,7 @@ export const GridRow = (props: GridRowProps) => {
   const rowClassName =
     isFunction(options.getRowClassName) &&
     options.getRowClassName(apiRef!.current.getRowParams(id));
-  const cssClasses = classnames(GRID_ROW_CSS_CLASS, className, rowClassName, options.classes?.row, {
+  const cssClasses = clsx(GRID_ROW_CSS_CLASS, className, rowClassName, options.classes?.row, {
```

Comment:
I think it's better to extract the props that we need from options as a pattern. 
This is to avoid having`[options]` as a dependency as it will look at all options, instead of looking at `options.something`...
Small detail :)

Reply:
Done ;)